src/components/TldrawCanvas.tsx

```diff
@@ -450,7 +450,7 @@ const COLOR_PALETTE: Record<string, string> = {
   white: "#ffffff",
   yellow: "#ffc078",
 };
-const COLOR_ARRAY = Array.from(TL_COLOR_TYPES);
+const COLOR_ARRAY = Array.from(TL_COLOR_TYPES).reverse();
```

Comment:
reversed so that the initial custom discourse nodes created by users do not default to black/grey